### PR TITLE
Glossary version for Portuguese

### PR DESCRIPTION
**Pull request by pragmaticivan**

### CODE REVIEW

File: GLOSSARY.md [hunk 1]
```diff
@@ -45,6 +45,48 @@ Class names and proper nouns do not get translated.
...
+- interpolation - interpolação
+- list - lista
+- map - mapa
+- metaprogramming - metaprogramação
+- module - módulo
+- pattern matching - correspondência de padrão
+- private (function) - privada (função)
+- protected (function) - protegida (função)
+- public (function) - pública (função)
+- rebind variable - religar variável
```
Comment (philss) [inline GLOSSARY.md]: I think `associar variável novamente` is more clear. What do you think?

File: GLOSSARY.md [hunk 2]
```diff
@@ -45,6 +45,48 @@ Class names and proper nouns do not get translated.
...
+- actors - atores
+- atom - átomo
+- function clauses - cláusulas de função
+- guard - guarda
+- guard operator - operador de guarda
+- keyword list - lista de palavras
+- match operator - operador para procura de padrões
+- pin operator - operator para apontar para último valor atribuido a variável
+- pipe operator - operador pipe
+- sigils - tag de padrão
```
Comment (philss) [inline GLOSSARY.md]: Not sure about this term translation. Sigils is a hard term to explain. They are used to transform the inner text into another type of data (like `~r/[0-9]/` that creates a Regex struct) or a different string representation (like `~s(this is a string with "double" quotes, not 'single' ones)` that escape the double quotes).

I would translate to something like: `operadores para transformação de strings em outras strings ou novos tipos de dados`. It's a bit longer but is more clear.

What do you think?

File: GLOSSARY.md (hunk 1 above)
Comment (pragmaticivan) [inline GLOSSARY.md]: I agree.

File: GLOSSARY.md (hunk 2 above)
Comment (pragmaticivan) [inline GLOSSARY.md]: :+1: , the official documentation says: `sigils are one of the mechanisms provided by the language for working with textual representations`

I would say: `mecanismos para transformação de representações textuais` Once it's created to literal representations.

Comment (philss) [inline GLOSSARY.md]: Cool, I agree. :+1:

**Comment by pragmaticivan** [comment]
@pjhampton @doomspork @brain-geek I also fixed a typo on `subtraction`. Please review this one.

**Comment by doomspork** [comment]
@pragmaticivan looks good to me.  Would you mind adding another change for me while you're at it?  

I think a table of contents at the top of this file would make it easier on translators to get to their language of choice.  What do you think?

**Comment by doomspork** [comment]
Hey @pragmaticivan I think this needs to be rebased on master.  What do you think about my previous comment too?

**Comment by pragmaticivan** [comment]
Hey,  I'm in a trip in chicago and will continue that tomorrow.  Thanks
On Apr 2, 2016 8:30 PM, "Sean Callan" <notifications@github.com> wrote:
>
> Hey @pragmaticivan I think this needs to be rebased on master. What do
you think about my previous comment too?

**Comment by doomspork** [comment]
No worries, enjoy your trip @pragmaticivan :grinning:

**Comment by pragmaticivan** [comment]
@doomspork can you review again? Thanks, just updated with everything we need.

**Comment by doomspork** [comment]
Sure thing @pragmaticivan, I'll make a point to look through this today.

**Comment by doomspork** [comment]
Looks good @pragmaticivan, thanks!  :+1:

**Pull request merged**